app/services/onlinetesting/phase3/EvaluatePhase3ResultService.scala

```diff
@@ -64,6 +65,13 @@ trait EvaluatePhase3ResultService extends EvaluateOnlineTestResultService[Phase3
       case _ => throw new IllegalStateException(s"Illegal number of phase3 active tests with results " +
         s"for this application: ${application.applicationId}")
     }
-    savePassMarkEvaluation(application, schemeResults, passmark)
+
+    CurrentSchemeStatusHelper.getSdipResults(application).map { sdip =>
+      if (application.isSdipFaststream) {
```

Comment:
Don't you already at this point, the application is sdip faststream?

I think if it is not sdipFastStream, function getSdipResults will return None.

Reply:
See comment above, Miguel